### DISCUSSION TITLE
Welcome to JanusXR Discussions!

### DISCUSSION THREAD

**Discussion by jbaicoianu** [jbaicoianu/janusweb#239] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lbgrid** [comment]
Hello JanusWeb Community!
Governor Littlebird here. Lead Developer and Governor for Littlebird Metaverse. I use JanusWeb for lots of cool things, but most importantly, it provides me a virtual world in which i reside full time, and the means to provide this for others. Love to help you with anything, feel free to ask me or anyone here.
